projects/samples/howto/controllers/vision/Makefile

```diff
@@ -27,7 +27,7 @@ USE_C_API = true
 ifeq ($(OSTYPE),windows)
   OPENCV_VERSION := $(subst .,,$(shell opencv_version))
   INCLUDE = -I/mingw64/include/opencv4
-  LIBRARIES = -L"$(WEBOTS_HOME)/msys64/mingw64/bin" -L/mingw64/bin -lopencv_core$(OPENCV_VERSION) -lopencv_imgproc$(OPENCV_VERSION)
+  LIBRARIES = -L/mingw64/bin -lopencv_core$(OPENCV_VERSION) -lopencv_imgproc$(OPENCV_VERSION)
```

Comment:
I can't neither run or recompile this controller from the release (both works fine from the development environement):
```
make
make: opencv_version: Command not found
# updating vision.d
make: opencv_version: Command not found
# compiling vision.cpp
vision.cpp:25:10: fatal error: opencv2/core/core.hpp: No such file or directory
 #include <opencv2/core/core.hpp>
          ^~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [C:/Program Files/Webots/resources/Makefile.include:691: build/release/vision.o] Error 1
```
![capture](https://user-images.githubusercontent.com/2461619/52554839-66b03f80-2de8-11e9-83a5-2c26cdca84a9.PNG)